### DISCUSSION TITLE
Flutter - Concevoir ses propres snippets sur Visual studio code

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#176] [Announcements]
# Flutter - Concevoir ses propres snippets sur Visual studio code

Coder rapidement en concevant vos propres snippets Dart dans vos projets Flutter.

https://zerofiltre.tech/articles/254

**Comment by imphilippesimo** [comment]
> 3/ Forme d’un Snippet Dart    

Je préfère **Structure d'un snippet dart**  
De plus pas besoin de donner des détails sur du Json.  
Enfin, en lieu et place d'un JSON quelconque, illustre plustôt un  un snippet dart.
